.env.development

```diff
@@ -1,4 +1,4 @@
-NEXT_PUBLIC_AUTH_DOMAIN="http://egghead.af:5000"
+NEXT_PUBLIC_AUTH_DOMAIN="https://app.egghead.io"
```

Comment:
Looks good! But this PR shouldn't change the `.env.development` variables.

Reply:
ah! Good catch!